audio_clip/model/clip/clip.py

```diff
@@ -158,9 +158,6 @@ def patch_float(module):
 
         model.float()
 
-    global _tokenizer
```

Comment:
Did we remove this because we don't need the language model?

Reply:
For now we are only doing audio here